fs/cachefs/cache.go

```diff
@@ -118,6 +118,7 @@ func (c *Cache) getEntries(ctx context.Context, id string, expirationTime time.D
 	}
 
 	c.mu.Lock()
+	defer c.mu.Unlock()
```

Comment:
`c.mu.Unlock()` was missing at line 131 below.

Reply:
good catch, wondering why I did it like that.